app/controllers/projects_controller.rb

```diff
@@ -65,6 +65,13 @@ def destroy
     end
   end
 
+  def sort
```

Comment:
might we perhaps prefer a RESTful endpoint here?  using an update endpoint to change things.  There's an implicit collection of projects here, perhaps there should be a new model called ProjectCollection that has it's own controller that we can submit updates to ...